ch04packaging/03Packaging.ipynb.py

```diff
@@ -599,19 +646,24 @@ def test_greeter(fixture):
 
 # %% [markdown]
 # Finally, we typically don't want to include the tests when we distribute our software for our users.
-# We can make sure they are not included using the `exclude` option on when telling `setuptools` to find packages.
+# We can make sure they are not included using the `exclude` option on when telling `hatch` to find packages.
```

Comment:
Do we need the `exclude` option if the tests are not under the package folder?

Reply:
Ah, not really, I can exclude the `exclude`

Reply:
`exclude` exclude to exclude the exclude.... [buffalo buffalo buffalo...](https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo)

Reply:
😂